catboost/libs/metrics/stress-test/fairloss_stress_text.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
all these test are not going be run in our CI

you should write a test using `pytest` framework, see `util/tests/cython`, `catboost/pytest` or `library/python/strings` for examples